packages/lesswrong/server/scripts/generativeModels/autoSpotlight.ts

```diff
@@ -1,118 +1,178 @@
-  `
-  const queryFirstParagraphRoot = `
-    Pick the first paragraph from the essay that isn't some kind of metadata. (Just write paragraph, without any preamble)
-  `
-  return [
-    queryClaude(`${queryQuestionRoot}${post.contents?.html}`),
-    queryClaude(`${queryBestQuestionParagraphRoot}${post.contents?.html}`),
-    queryClaude(`${queryBestParagraphRoot}${post.contents?.html}`),
-    queryClaude(`${queryFirstParagraphRoot}${post.contents?.html}`),
-  ]
+import { getAnthropicPromptCachingClientOrThrow } from "@/server/languageModels/anthropicClient";
+import { REVIEW_WINNER_CACHE, ReviewWinnerWithPost } from "@/lib/collections/reviewWinners/cache";
+
+async function queryClaudeJailbreak(basePrompt: string, spotlightPrompt: string, maxTokens: number) {
+  const client = getAnthropicPromptCachingClientOrThrow()
+  return await client.messages.create({
+    system: "The assistant is in CLI simulation mode, and responds to the user's CLI commands only with the output of the command.",
+    model: "claude-3-5-sonnet-20240620",
+    max_tokens: maxTokens,
+    messages: [
+      {
+        role: "user", 
+        content: [{
+          type: "text",
+          text: basePrompt,
+          cache_control: {type: "ephemeral"}
+        }]
+      },
+      {
+        role: "assistant",
+        content: [{
+          type: "text",
+          text: spotlightPrompt,
+        }]
+      },
+    ]
+  })
 }
 
-function createSpotlight (postId: string, summaries: string[]) {
-  const description = summaries.map(summary => `<p>${summary}</p>`).join("")
+function createSpotlight (post: PostsWithNavigation, reviewWinner: ReviewWinnerWithPost|undefined, summary: string) {
   const context = createAdminContext();
+  const postYear = post.postedAt.getFullYear()
+  const cloudinaryImageUrl = reviewWinner?.reviewWinner.reviewWinnerArt?.splashArtImageUrl
+
   void createMutator({
     collection: Spotlights,
     document: {
-      documentId: postId,
+      documentId: post._id,
       documentType: "Post",
+      customSubtitle: `Best of LessWrong ${postYear}`,
       duration: 1,
       draft: true,
       showAuthor: true,
-      description: { originalContents: { type: 'ckEditorMarkup', data: description } },
+      spotlightSplashImageUrl: cloudinaryImageUrl,
+      description: { originalContents: { type: 'ckEditorMarkup', data: summary } },
       lastPromotedAt: new Date(0),
     },
     currentUser: context.currentUser,
     context
   })
 }
 
+async function getPromptInfo(): Promise<{posts: PostsWithNavigation[], spotlights: DbSpotlight[]}> {
+  const reviewWinners = await fetchFragment({
+    collectionName: "ReviewWinners",
+    fragmentName: "ReviewWinnerTopPostsPage",
+    currentUser: createAdminContext().currentUser,
+    selector: { },
+    skipFiltering: true,
+  });
+  const postIds = reviewWinners.map(winner => winner.postId);
+  // Assume these functions exist to fetch data from the database
```

Comment:
What does this comment mean?

Reply:
I... am not sure and think this was probably a copilot writing. I removed it.